spec/helpers/capybara_poltergeist_screenshot.rb

```diff
@@ -0,0 +1,12 @@
+require 'launchy'
+
+module Capybara
+  module PoltergeistScreenshot
+    def screenshot_and_open_image(full: false)
```

Comment:
Yep, I learned something new! I've pulled out the helper and launchy reference in the Gemfile